### ISSUE TITLE
Screen goes black

### ISSUE THREAD

**Issue by karmette** [xMrVizzy/Skyblocker#11]
Don't know if its a skyblocker issue or a different mod issue,
My screen turned black (first screenshot) after i did /warp and closed out of the warp menu

screenshots:
![image](https://user-images.githubusercontent.com/48113149/118984458-5fb2f880-b932-11eb-9116-05fd4cea067b.png)
![image](https://user-images.githubusercontent.com/48113149/118984478-6477ac80-b932-11eb-9190-69c5cd9f3197.png)
![image](https://user-images.githubusercontent.com/48113149/118984498-680b3380-b932-11eb-853f-f845d46b9e26.png)
![image](https://user-images.githubusercontent.com/48113149/118984513-6b9eba80-b932-11eb-8155-6b68f775ef0f.png)
This is after the bug happened
NOT my texture pack
I restarted my game and it looked fine
![image](https://user-images.githubusercontent.com/48113149/118985001-ee277a00-b932-11eb-8958-a966a091d01c.png)
![image](https://user-images.githubusercontent.com/48113149/118985325-347cd900-b933-11eb-969f-bf231773d468.png)
![image](https://user-images.githubusercontent.com/48113149/118985399-452d4f00-b933-11eb-9730-793d02e3cd60.png)
As a test, I press E and close out, no black screen.
![image](https://user-images.githubusercontent.com/48113149/118985464-51b1a780-b933-11eb-8ead-546e500f4958.png)
I test by doing /warp and closing out the /warp menu
![image](https://user-images.githubusercontent.com/48113149/118985684-84f43680-b933-11eb-96ee-d6e404237bf5.png)
There's no black screen, so so far i don't know how to replicate the bug.
However, if it happens again I'll come back here.
i unfortunately don't have the log right now, but if it happens again I'll send it here

**Comment by Zailer43** [comment]
I think this is related to [MC-4647: World turns completely black when having night vision, and being in the void or having blindness](https://bugs.mojang.com/browse/MC-4647)

**Comment by karmette** [comment]
well, its similar, but my text goes black and my hotbar does too.

**Comment by karmette** [comment]
Yes, i did have the night vision effect but i dont remember having blindness

**Comment by MaxBr5** [comment]
<pre>Do you use optifine for optifine is forbidden and
you'd be better off with <a href="http://github.com/CaffeineMC/lithium-fabric">lithium</a> and <a href="http://github.com/CaffeineMC/phosphor-fabric">phosphor</a>
and if you want shaders, <a href=https://github.com/IrisShaders/Iris>iris</a>.

**Comment by MaxBr5** [comment]
<pre>Optifine has infinite incompatibilities
and is more like Optinotsofine.</pre>
